VideoLocker/src/test/java/org/edx/mobile/view/BaseFragmentActivityTest.java

```diff
@@ -500,67 +487,6 @@ protected Intent assertNextStartedActivityForResult(
     }
 
     /**
-     * Testing download progress menu visibility states and click behaviour
-     * (starting DownloadActivity). Only when both AppConstants.offline_flag
-     * is true and there is a downloading entry in the database, should the
-     * progress bar be visible.
-     */
-    @Test
-    public void downloadProgressViewTest() {
```

Comment:
This test should not have been removed, as it only tests the action bar indicator. The override in `CourseBaseActivityTest` will also need to be reverted.